common/sampling.h

```diff
@@ -14,7 +14,7 @@ typedef struct llama_sampling_params {
     int32_t n_probs           = 0;     // if greater than 0, output the probabilities of top n_probs tokens.
     int32_t top_k             = 40;    // <= 0 to use vocab size
     float   top_p             = 0.95f; // 1.0 = disabled
-    float   min_p             = 1.0f; // 1.0 (or 0.0) = disabled
+    float   min_p             = 0.0f; // 1.0 (or 0.0) = disabled
```

Comment:
```plaintext
 //
 //
//
 //
 //
```

Our illustrious leader likes it when stuff lines up nicely.

Reply:
Before I commit is the whitespace fixed or nah?